volume/volume.go

```diff
@@ -270,7 +270,6 @@ func ParseMountSpec(cfg mounttypes.Mount, options ...func(*validateOpts)) (*Moun
 		}
 		mp.CopyData = DefaultCopyMode
 
-		mp.Driver = DefaultDriverName
```

Comment:
It must be removed otherwise we end up with "local" set when we don't want it to be.
It gets set after volume creation using `v.DriverName()`